sqlserver/assets/configuration/spec.yaml

```diff
@@ -842,6 +842,19 @@ files:
             type: boolean
             example: false
             display_default: false
+    - name: collect_raw_query_statement
+      hidden: true
+      description: |
+        Configure the collection of raw query statements in query activity and execution plans.
+        Note: This option only applies when `dbm` is enabled.
```

Comment:
Is "raw query statements" enough to warn customers that the agent might emit sensitive information, or should we make it more explicit?